File: app/src/main/java/wyyu/multi/cell_integer/data/DataIntegerB.java
package wyyu.multi.cell_integer.data;

import wyyu.multi.app.Constants;

/**
 * Created by wyyu on 2020-05-07.
 **/
public class DataIntegerB implements IIntegerData {

    public int index;

    public int resId;

    public String content;

    public DataIntegerB(int index) {
        this.index = index;
        this.resId = Constants.RES_ID;
        this.content = Constants.CONTENT;
    }

    @Override public int loadDataType() {
        return IIntegerData.DATA_B;
    }
}

File: app/src/main/java/com/wyyu/multiktest/cell_Integer/data/DataIntegerB.kt
package com.wyyu.multiktest.cell_Integer.data

import com.wyyu.multiktest.app.Constants

class DataIntegerB(
    val index: Int,
    val resId: Int = Constants.RES_ID,
    val content: String = Constants.CONTENT
) : IDataInteger {

    override fun dataType(): Int {
        return IDataInteger.DATA_B
    }
}